### PR TITLE
qdirstat: new recipe

### PR DESCRIPTION
**Pull request by return**
A Qt based directory statistics application, based on KDirStat. Tested on x86.

### CODE REVIEW

File: sys-apps/qdirstat/qdirstat-1.2.recipe [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+HOMEPAGE="https://github.com/shundhammer/qdirstat"
+COPYRIGHT="2015, 2017 Stefan Hundhammer"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/shundhammer/qdirstat/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="d79734915768ae858c784142c7a07b7599178329b40c73894bf575decaf1a911"
+PATCHES="qdirstat-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86 ?x86_64"
```
Comment (korli) [inline sys-apps/qdirstat/qdirstat-1.2.recipe]: x86_64 can't be a secondary arch.

File: sys-apps/qdirstat/qdirstat-1.2.recipe [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	"
+BUILD()
+{
+	qmake .
+	make $jobArgs
+}
+INSTALL()
+{
+	mkdir -p $appsDir/QDirStat
+	cp src/qdirstat $appsDir/QDirStat/QDirStat
```
Comment (korli) [inline sys-apps/qdirstat/qdirstat-1.2.recipe]: Please just install the binary in $appsDir

Comment (return) [inline sys-apps/qdirstat/qdirstat-1.2.recipe]: I have basically renamed the binary to start-cases whilst copying it to its folder. Is this discouraged in the recipe?

Comment (korli) [inline sys-apps/qdirstat/qdirstat-1.2.recipe]: A folder for a single binary is discouraged. Just copy the binary in $appsDir, not a subdirectory.

**Pull request merged**